modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -284,6 +284,20 @@ definition {
 		PortletEntry.save();
 	}
 
+	macro addObjectFolderViaUi {
+		Click(locator1 = "ObjectAdmin#ADD_OBJECT_FOLDER");
+
+		Type(
+			locator1 = "ObjectAdmin#ADD_FOLDER_LABEL",
+			value1 = ${fieldLabel});
+
+		Type(
+			locator1 = "ObjectAdmin#DETAILS_NAME_CUSTOM_OBJECT",
+			value1 = ${fieldName});
+
+		Click(locator1 = "ObjectAdmin#CREATE_FOLDER_BUTTON");
```

Comment:
@thaiscabrals hey Thais, have we tried existing and generic paths before creating new ones?